examples/skysync-cli-example-create-job-01.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+.DESCRIPTION
+    Sample script in PowerShell to showcase the use of the CLI to generate and manage SkySync jobs. Example scenario:
+        * Create an NFS connection
+        * Create a Box connection
+        * Create a SkySync job that utilizes these connections
+        * Start the job
+
+.REFERENCE
+    https://skysync.atlassian.net/wiki/spaces/SSP/pages/15302683/SkySync+Command-line+Interface
```

Comment:
This isn't a public link

Reply:
replaced with github repo link for public consumption